crates/util/src/paths.rs

```diff
@@ -8,17 +8,29 @@ use serde::{Deserialize, Serialize};
 
 lazy_static::lazy_static! {
     pub static ref HOME: PathBuf = dirs::home_dir().expect("failed to determine home directory");
-    pub static ref CONFIG_DIR: PathBuf = HOME.join(".config").join("zed");
+    pub static ref CONFIG_DIR: PathBuf = if cfg!(target_os = "windows") {
+        dirs::config_dir()
+            .expect("failed to determine config directory")
+            .join("Zed")
+    } else {
+        HOME.join(".config").join("zed")
+    };
     pub static ref CONVERSATIONS_DIR: PathBuf = CONFIG_DIR.join("conversations");
     pub static ref EMBEDDINGS_DIR: PathBuf = CONFIG_DIR.join("embeddings");
     pub static ref THEMES_DIR: PathBuf = CONFIG_DIR.join("themes");
     pub static ref LOGS_DIR: PathBuf = if cfg!(target_os = "macos") {
         HOME.join("Library/Logs/Zed")
+    } else if cfg!(target_os = "windows") {
+        dirs::data_local_dir()
+            .expect("failed to determine LocalAppData directory")
+            .join("Zed/logs")
     } else {
         CONFIG_DIR.join("logs")
     };
-    pub static ref SUPPORT_DIR: PathBuf = if cfg!(target_os = "macos") {
-        HOME.join("Library/Application Support/Zed")
+    pub static ref SUPPORT_DIR: PathBuf = if cfg!(any(target_os = "macos", target_os = "windows")) {
```

Comment:
Can we change this to `if cfg!(target_os = "macos") {... } else cfg!(target_os = "windows") { ... } else {...}`, and restore the original `HOME.join` call?

Reply:
Reverted it